applications/app/controllers/SurveyPageController.scala

```diff
@@ -26,8 +26,8 @@ class SurveyPageController(wsClient: WSClient) extends Controller with Execution
       Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.myDigest404(StaticPages.simpleSurveyStaticPageForId(request.path))))
    }
 
-  def renderWeekendReading404Page() = Action { implicit request =>
-      Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.weekendReading404(StaticPages.simpleSurveyStaticPageForId(request.path))))
+  def renderWeekendReadingPage() = Action { implicit request =>
+      Cached(defaultCacheDuration)(RevalidatableResult.Ok(views.html.survey.weekendReading(StaticPages.simpleSurveyStaticPageForId(request.path))))
```

Comment:
Since this isn't a survey anymore, it would better to separate it out from the survey stuff in some way. As well as being clearer for developers, currently there are some things the user will see that are survey-related, e.g. [this code](https://github.com/guardian/frontend/blob/b1d0268efd0fd989dcdf5fabb3e566072012d02b/common/app/staticpages/StaticPages.scala#L10) will make the page title "Guardian Survey Page".